lib/serverrequestcontext.go

```diff
@@ -188,11 +188,13 @@ func (ctx *serverRequestContextImpl) verifyX509Token(ca *CA, authHdr, method, ur
 	if err2 != nil {
 		return "", caerrors.NewAuthenticationErr(caerrors.ErrInvalidToken, "Invalid token in authorization header: %s", err2)
 	}
+
 	// Make sure the caller's cert was issued by this CA
-	err2 = ca.VerifyCertificate(cert)
+	err2 = ca.VerifyCertificate(cert, (ctx.endpoint.Path == "reenroll" && ctx.ca.Config.CA.IgnoreCertExpiry))
```

Comment:
While this works I find it difficult to read when too much is done in a single line.
How about setting a variable whose name matches the parameter name prior to the function call and then passing the variable, e.g.

```
ignoreCertExpiry := ctx.endpoint.Path == "reenroll" && ctx.ca.Config.CA.IgnoreReenrollCertExpiry
err2 = ca.VerifyCertificate(cert,ignoreCertExpiry)
```

Reply:
ah yes.. it is used twice... will pull it out into it's own variable